src/main/java/de/dbtool/cli/subcommands/ListProfilesCommand.java

```diff
@@ -13,6 +25,39 @@ public class ListProfilesCommand implements Runnable {
 
     @Override
     public void run() {
-        // TODO: Implement this
+        Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
+        Set<ConstraintViolation<ListProfilesCommand>> violations = validator.validate(this);
+
+        if(!violations.isEmpty()) {
+            StringBuilder errorMsg = new StringBuilder();
+            for(ConstraintViolation<ListProfilesCommand> violation : violations) {
+                errorMsg.append("ERROR: ").append(violation.getMessage()).append("\n");
+            }
+            System.err.println(errorMsg);
+            return;
+        }
+
+        try {
+            ProfileHandler profileHandler = new ProfileHandler();
+            TablePrinter tablePrinter = new TablePrinter(12);
+
+            Profile[] profiles = profileHandler.listProfiles();
+            if(profiles == null || profiles.length == 0) {
+                System.out.println("No profiles found!");
+                return;
+            }
+            List<String[]> tableData = new ArrayList<>(profiles.length);
+            tableData.add(new String[] {"Name", "Hostname", "Port", "Database", "Username", "Password", "Type", "Driver"});
+            for(Profile p : profiles) {
+                tableData.add(new String[] {p.name, p.hostname, Integer.toString(p.port), p.dbName, p.username, p.password, p.type.toString(), p.driverPath});
```

Comment:
Please print (*) instead of the password 😄

Reply:
Haha good point (= *) lol. 💯 
Thought about that but forgot it at the end ...

Reply:
![grafik](https://github.com/mad201802/db-tool/assets/44781682/207b29e7-061b-45fd-a5ef-c7c6b9542762)